plugins/services/src/js/components/forms/GeneralServiceFormSection.js

```diff
@@ -126,23 +124,43 @@ class GeneralServiceFormSection extends Component {
     }
 
     return (
-      <p>
-        {'Need to run a service with multiple containers? '}
-        <a className="clickable" onClick={this.handleOpenConvertToPodModal}>
-          Add another container
-        </a>.
-      </p>
+      <div className="pod pod-narrow pod-short">
```

Comment:
@leemunroe That will be fixed with the classes I provided above (specifically the `flush-horizontal` class).